reports/2022.html

```diff
@@ -159,15 +163,15 @@ <h3>Table of Contents</h3>
             </tr>
             <tr>
               <th><a href="#lazy-custom-element-definitions">Lazy custom element definitions</a></th>
+              <td><a href="https://github.com/WICG/webcomponents/issues/782">WICG/webcomponents#782</a></td>
               <td></td>
-              <td></td>
-              <td></td>
+              <td>Uncertain</td>
             </tr>
             <tr>
               <th><a href="#dom-parts">DOM Parts</a></th>
+              <td><a href="https://github.com/WICG/webcomponents/blob/gh-pages/proposals/DOM-Parts.md">DOM Part API - First Step of Template Instantiation</a></td>
```

Comment:
@EisenbergEffect ,
DOM parts needs a link to discussion media. Like an issue in format suggested by @Westbrook.
I have created such for [DSD](https://github.com/w3c/webcomponents-cg/issues/32), we need similar for DOM parts.